### ISSUE TITLE
Resource usage history

### ISSUE THREAD

**Issue by oleg-kondaurov** [jenkinsci/lockable-resources-plugin#533]
### What feature do you want to see added?

Display the list of jobs, where the resource was used. It may be useful to easily understand if the possible resource outage is the reason of job fail, as some other jobs also failed.

For example, the `Credentials` plugin has something similar:
![image](https://github.com/jenkinsci/lockable-resources-plugin/assets/47348488/f76d744d-081b-4a79-8f97-ef7e5c6cfb63)

**Comment by mPokornyETM** [comment]
Nice idea. Can be combined with #393 
But this feature takes longer then one hot summer night. Maybe good starts for newcommers.

**Comment by mPokornyETM** [comment]
And maybe we can implement this too ;-) #320

**Comment by oleg-kondaurov** [comment]
@mPokornyETM Could you help to assign the issue on @AniketNS please?

**Comment by mPokornyETM** [comment]
@AniketNS , when you need help to start here, contact me. I will try to answer ASAP.

**Comment by AniketNS** [comment]
I'm just going to start working on this issue, Sir. Sorry for the delay. I'll be troubling you, whenever I need help.

**Comment by mPokornyETM** [comment]
one hit
you need to do like this
https://github.com/jenkinsci/lockable-resources-plugin/blob/c9db3ae76e3cccaf6f563e571fc87da2787155f3/src/main/java/org/jenkins/plugins/lockableresources/queue/LockRunListener.java#L60C2-L61C39

in 
https://github.com/jenkinsci/lockable-resources-plugin/blob/c9db3ae76e3cccaf6f563e571fc87da2787155f3/src/main/java/org/jenkins/plugins/lockableresources/LockStepExecution.java#L114C4-L114C30

Then it shall works ;-)
Maybe we shall change the page design. Because it looks very weird. But that "only" my personal UX

**Comment by AniketNS** [comment]
Hello @mPokornyETM, I looked at the codes from the above links. But I didn't understand anything from it. Can you please guide me a little on what I should do?

**Comment by mPokornyETM** [comment]
@oleg-kondaurov @AniketNS can somebody review this #583 changes pls? It does not fully implement your request. If ound it by implementation and want to fixed in partly PR. Otherwise it will be egain to big PR for review. Thx

**Comment by AniketNS** [comment]
Hello, I didn't get this issue very well and had no idea how to solve this or if this issue is already solved in any other issue. so I'm unassigning from this issue.

**Comment by kimchidonut0** [comment]
Hi @mPokornyETM, I am interested in working on this issue. Could you please assign this to me?

**Comment by kimchidonut0** [comment]
Thanks @mPokornyETM.

I was wondering if you could help clarify on the task a bit please. I saw that earlier there was a PR [583](https://github.com/jenkinsci/lockable-resources-plugin/pull/583) that shows used resource for individual pipeline and freestyle jobs. What should the next step be? Would it be something like creating a centralized display for usages of all lockable resources?

**Comment by mPokornyETM** [comment]
yes, an extra page per resource, where will be possible to see the history. Also an page (all sub-tab in main page) where we see the history over all resources will be fine. Like timelines. To see how long it was allocated and how long are other "jobs" waiting.
This might take a liitle bit longer.
